committee-code-of-conduct/governance/onboarding-offboarding.md

```diff
@@ -23,17 +23,18 @@ Different actions on this list must be carried out by different members:
 
 ### Kubernetes/community permissions and google permissions
 
-**Who executes:** Carryover members should ensure that outgoing members are removed and incoming members are invited. 
+**Who executes:** Carryover members should ensure that outgoing members are removed and incoming members are invited.
 
-- [ ] Update `kubernetes/community` [`sigs.yaml`](https://github.com/kubernetes/community/blob/master/sigs.yaml) 
-- [ ] Update `kubernetes/k8s.io` [`groups.yaml`](https://github.com/kubernetes/k8s.io/blob/main/groups/groups.yaml)
+- [ ] Update `kubernetes/community` [`sigs.yaml`](/sigs.yaml)
+- [ ] Update `kubernetes/k8s.io` [`groups.yaml`](https://git.k8s.io/k8s.io/groups/committee-code-of-conduct/groups.yaml)
```

Comment:
I think the add of the `groups/committee-code-of-conduct/groups.yaml` check/update is definitely needed, but also `groups/groups.yaml` check/update should remain.

Reply:
> I think the add of the `groups/committee-code-of-conduct/groups.yaml` check/update is definitely needed, but also `groups/groups.yaml` check/update should remain.

@tpepper -- The CoCC and Steering were moved into subdirectories in https://github.com/kubernetes/k8s.io/pull/1906, so updates to the top-level `groups/groups.yaml` are no longer required.

That said, you've made me realize that the docs do not include updates to OWNERS_ALIASES for k/k8s.io, so I'll add that.

Reply:
> the docs do not include updates to OWNERS_ALIASES for k/k8s.io

Added in https://github.com/kubernetes/community/pull/6216/commits/89096ea7d82dd2836a4f50d911b0ba9322726174.